tests/ui/return/tail-expr-as-potential-return.rs

```diff
@@ -45,3 +45,19 @@ async fn foo2() -> i32 {
     }
     0
 }
+
+struct Receiver;
+impl Receiver {
+    fn generic<T>(self) -> Option<T> {
+        None
+    }
+}
+fn method() -> Option<i32> {
+    if true {
+        Receiver.generic();
+        //~^ ERROR type annotations needed
+        //| HELP you might have meant to return this value
```

Comment:
Opened https://github.com/rust-lang/rust/issues/127155 so I don't forgor about this.

Reply:
Yeah that was my typo from copypasting from the file without thinking rather than just writing my own UI test -- `//|` is definitely not valid and I should have seen that lmao